src/ui/ProjectsFolderBrowser.cpp

```diff
@@ -815,6 +815,14 @@ ProjectsFolderBrowser::ProjectByPath(const BString& fullPath) const
 	return nullptr;
 }
 
+void
+ProjectsFolderBrowser::SelectAndScroll(ProjectFolder* projectFolder)
+{
+	Select(IndexOf(GetProjectItemForProject(projectFolder)));
```

Comment:
I would check for nullptr here, before using IndexOf() (and just return without doing anything in that case)